Tests/VB/StatementTests.cs

```diff
@@ -755,6 +755,90 @@ void TestMethod()
     End Sub
 End Class");
         }
+        [Fact(Skip = "month As Integer is reduced by Simplifier, and month threated as Date.Month method")]
```

Comment:
I think this is likely  caused by the fact that the converter currently puts a bunch of Imports guesses directly into the file which probably makes the symbol ambiguous in the initial parsing.

Reply:
It is because of type promotion. There is the DateAndTime module in Microsoft.VisualBasic namespace, and its members are promoted to Microsoft.VisualBasic namespace. Microsoft.VisualBasic namespace  seems to imported implicitly. But the same problem can occur with any module if it's namespace is imported.

Reply:
> Microsoft.VisualBasic namespace seems to imported implicitly

Exactly - all of these get put at the top of every file:
https://github.com/icsharpcode/CodeConverter/blob/master/ICSharpCode.CodeConverter/Util/VisualBasicCompiler.cs#L71